src/views/Home/HomeView.vue

```diff
@@ -97,6 +97,10 @@
     name: yup
       .string()
       .required('Full Name is a required field')
+      .test('non-numeric', 'Full Name must contain only alphabetic characters', (value) => {
```

Comment:
Should this be a part of this PR? It looks like it's out of scope

Reply:
Isso foi algo que fiz a parte para subir em outro pr que eu fosse subir.